### PR TITLE
 If specified, posts event data to URL upon instance interruption action. 

### PR DESCRIPTION
**Pull request by voriol**
New attempt to fix: https://github.com/kubernetes/kops/issues/14561

(From: https://github.com/kubernetes/kops/pull/14562)

### CODE REVIEW

**Comment by hakman** [comment]
> Adding label `do-not-merge/contains-merge-commits` because PR contains merge commits, which are not allowed in this repository. Use `git rebase` to reapply your commits on top of the target branch. Detailed instructions for doing so can be found [here](https://git.k8s.io/community/contributors/guide/github-workflow.md#4-keep-your-branch-in-sync).

@voriol Please rebase.

**Comment by hakman** [comment]
@voriol See commit https://github.com/kubernetes/kops/pull/16009/commits/9b479b7ed811cbd3b2515c0767c2f96c6324b1b0 in https://github.com/kubernetes/kops/pull/16009/commits.

**Comment by voriol** [comment]
I have reviewed the test output and I can't quite see where the problem is, I think it's not related to my changes. Can you help me please?

Thank you

**Review by hakman — CHANGES_REQUESTED** [review]

**Comment by hakman** [comment]
> have reviewed the test output and I can't quite see where the problem is, I think it's not related to my changes.

It's probably a flake, please ignore for now.

File: docs/releases/1.28-NOTES.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 * Node Termination Handler is now enabled by default.
 
+* Posts event data to URL upon instance interruption action in aws-node-termination-handler with `WEBHOOK_URL`.
+
```
[the remark this answers is not in the document]
Comment (voriol) [reply docs/releases/1.28-NOTES.md]: moved! thanks

**Comment by voriol** [comment]
Hello @hakman , 

Is there any pending task on my part or is it pending your review?

Thank you

**Comment by hakman** [comment]
Hey @voriol, I will try to take a look this week, sorry for the delay.

**Comment by voriol** [comment]
Hello @hakman,

What do you want to do with this topic? Do we close it or continue it?

Thank You

**Pull request merged**